### ISSUE TITLE
store/memory/memstore.c:567:1: error: control reaches end of non-void function [-Werror=return-type]

### ISSUE THREAD

**Issue by brunoqc** [slact/nchan#230]
nchan 1.0.0 doesn't build on Alpine with this [Dockerfile](https://github.com/brunoqc/docker-nginx/blob/nchan/mainline/alpine/Dockerfile).

```
cc -c -pipe  -O -W -Wall -Wpointer-arith -Wno-unused-parameter -Werror -g   -I src/core -I src/event -I src/event/modules -I src/os/unix -I src/http/modules/perl -I /usr/include/libxml2 -I objs -I src/http -I src/http/modules -I src/http/v2 -I /tmp/nchan-1.0.0/src -I src/mail -I src/stream \
	-o objs/addon/memory/ipc-handlers.o \
	/tmp/nchan-1.0.0/src/store/memory/ipc-handlers.c
cc -c -pipe  -O -W -Wall -Wpointer-arith -Wno-unused-parameter -Werror -g   -I src/core -I src/event -I src/event/modules -I src/os/unix -I src/http/modules/perl -I /usr/include/libxml2 -I objs -I src/http -I src/http/modules -I src/http/v2 -I /tmp/nchan-1.0.0/src -I src/mail -I src/stream \
	-o objs/addon/memory/memstore.o \
	/tmp/nchan-1.0.0/src/store/memory/memstore.c
/tmp/nchan-1.0.0/src/store/memory/memstore.c: In function 'nchan_get_used_shmem':
/tmp/nchan-1.0.0/src/store/memory/memstore.c:567:1: error: control reaches end of non-void function [-Werror=return-type]
 }
 ^
cc1: all warnings being treated as errors
objs/Makefile:2084: recipe for target 'objs/addon/memory/memstore.o' failed
make[1]: Leaving directory '/usr/src/nginx-1.11.3'
make[1]: *** [objs/addon/memory/memstore.o] Error 1
make: *** [build] Error 2
Makefile:8: recipe for target 'build' failed
```

**Comment by slact** [comment]
What's your GCC version?

**Comment by brunoqc** [comment]
```
# gcc -v
Using built-in specs.
COLLECT_GCC=gcc
COLLECT_LTO_WRAPPER=/usr/libexec/gcc/x86_64-alpine-linux-musl/5.3.0/lto-wrapper
Target: x86_64-alpine-linux-musl
Configured with: /home/buildozer/aports/main/gcc/src/gcc-5.3.0/configure --prefix=/usr --mandir=/usr/share/man --infodir=/usr/share/info --build=x86_64-alpine-linux-musl --host=x86_64-alpine-linux-musl --target=x86_64-alpine-linux-musl --with-pkgversion='Alpine 5.3.0' --enable-checking=release --disable-fixed-point --disable-libstdcxx-pch --disable-multilib --disable-nls --disable-werror --disable-symvers --enable-__cxa_atexit --enable-esp --enable-cloog-backend --enable-languages=c,c++,objc,java,fortran,ada --disable-libssp --disable-libmudflap --disable-libsanitizer --enable-shared --enable-threads --enable-tls --with-system-zlib
Thread model: posix
gcc version 5.3.0 (Alpine 5.3.0)
```

**Comment by slact** [comment]
fixed in  3ef9fea.

**Comment by brunoqc** [comment]
It compiled. Thanks!
